lambdas/shared/t4_lambda_shared/utils.py

```diff
@@ -105,3 +105,30 @@ def buffer_s3response(s3response):
         raise IncompleteResultException("Error: Received an incomplete response from S3 Select.")
     response.seek(0)
     return response
+
+
+def call_s3_select(
+    s3_client: str,
```

Comment:
You could do only s3_client as positional, or s3_client, bucket, and key as positional, the rest should be required kwargs so:

```
s3_client: str,
*,
bucket
key,
etc,
```